lib/lists/helpers.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/**
+ * Get the format of the items in a list
+ * @param {array} item
+ * @returns {string}
+ */
+
+function getFormat(items) {
+  const tester = _.head(items);
+
+  if (_.isString(tester)) {
+    return 'STRING';
+  } else if (_.has(tester,'text')) {
+    return 'OBJECT';
+  } else {
+    log.error('List cannot be modified because it is not a simple list.', { action: 'modifyList' });
+  }
+}
+
+/**
+ * @param {array} items
+ * @param {string} testItem
```

Comment:
This says `testItem` is a string, but the code below says it may be an object (but no other type?)

Reply:
`testItem` can be either a string OR object, changing.